extensions/wikia/Recirculation/js/recirculation.js

```diff
@@ -42,36 +42,37 @@ require([
 					}
 				}
 			},
-			{
-				id: 'LI_impactFooter',
-				viewModule: 'ext.wikia.recirculation.views.impactFooter',
-				options: {
-					max: 9,
-					widget: 'wikia-impactfooter',
-					source: 'fandom',
-					opts: {
-						resultType: 'cross-domain',
-						domainType: 'fandom.wikia.com'
-					}
-				}
-			},
-			{
-				id: 'LI_footer',
-				viewModule: 'ext.wikia.recirculation.views.footer',
-				options: {
-					max: 3,
-					widget: 'wikia-footer-wiki-rec',
-					source: 'wiki',
-					title: 'Discover New Wikis',
-					width: 332,
-					height: 187,
-					flush: true,
-					opts: {
-						resultType: 'subdomain',
-						domainType: 'fandom.wikia.com'
-					}
-				}
-			}
+			// TODO - remove old recirculation footers
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1135)